### ISSUE TITLE
SpamAssasin -2.499	HDRS_MISSP	Misspaced headers

### ISSUE THREAD

**Issue by avreg7** [splitbrain/dokuwiki#3438]
Hello,
The mail-tester.com always showed 10/10 score.
Last two weeks stable 7.7 / 10 problem with SpamAssassin

SpamAssassin thinks you can improve
-2.499 HDRS_MISSP Misspaced headers

No changes were made to the  HTML version of text,  settings of SMTP and Accounts and others

can you help solve this issue?
Looking forward to hearing from you
Thanks,
Andrii

**Comment by mprins** [comment]
> can you help solve this issue?

Not really. Your report is missing key information for debugging and/or analysis. Please provide full mail headers.

All DW mail is sent through https://github.com/splitbrain/dokuwiki/blob/master/inc/Mailer.class.php

**Comment by avreg7** [comment]
-0.1 | DKIM_SIGNED | Message has a DKIM or DK signature, not necessarily validThis rule is automatically applied if your email contains a DKIM signature but other positive rules will also be added if your DKIM signature is valid. See immediately below.
-- | -- | --
0.1 | DKIM_VALID | Message has at least one valid DKIM or DK signatureGreat! Your signature is valid
0.1 | DKIM_VALID_AU | Message has a valid DKIM or DK signature from author's domainGreat! Your signature is valid and it's coming from your domain name
0.1 | DKIM_VALID_EF | Message has a valid DKIM or DK signature from envelope-from domain
-0.001 | FROM_EXCESS_BASE64 | From: base64 encoded unnecessarily
-2.499 | HDRS_MISSP | Misspaced headers
-0.001 | HTML_MESSAGE | HTML included in messageNo worry, that's expected if you send HTML emails
-0.001 | MIME_QP_LONG_LINE | Quoted-printable line longer than 76 chars
-0.001 | RCVD_IN_MSPIKE_H3 | Good reputation (+3)62.13.146.239 listed in wl.mailspike.net
-0.001 | RCVD_IN_MSPIKE_WL | Mailspike good senders
-0.001 | SPF_HELO_NONE | SPF: HELO does not publish an SPF Record
0.001 | SPF_PASS | SPF: sender matches SPF recordGreat! Your SPF is valid

**Comment by avreg7** [comment]
From : "Sarah Blythman (White Paper)" <sarah@wpcc.co.uk>
Bounce address : sarah@wpcc.co.uk
Reply-To : sarah@wpcc.co.uk

Dear Mr Gascoyne,

In case you missed this message last week, I had to get it out to you again.

This is the strongest conference on planning law and practice to be held yet; building on your knowledge, this year's advanced-level conference adopts a totally new focus.

Listen to solution-focused answers to 10 thorny questions.

These answers come from 7 supremely talented QCs, senior juniors and solicitors: all with first-hand experience of arguing the points directly.

Because of social distancing, we're going to do something unique.

An award-winning filmmaker will pre-record the talks. The footage will be edited and produced as a top-of-the-line production; the quality you would expect from a TV broadcast service.

We'll release the videos on Tuesday 11th May 2021 and keep them online for eighteen weeks only: "Planning Law: Shaping New Cases into Solution-Focused Advice for Your Clients."

This means you can watch on-demand, at your convenience, in chunks, pause, rewind and repeat watch for deeper understanding.

You'll get your 5 hrs CPD and not have to worry about social distancing, public transport or government pronouncements.

At only £119 plus VAT per person, or £250 plus VAT for your entire team, the value is exceptional (which includes your £30 thank you discount as thanks for your past support). There is no numbers limit on the team option.

https://whitepaper.co.uk/conferences/planning21pd

"Highly relevant for planning practice and the day to day issues faced by planners." (Shared by a past-attendee from Central Bedfordshire Council)

Here are the questions to be answered at the upcoming conference (plus speakers - all speaker quotes are taken from Chambers Directory and Legal 500):

** Conditions: What are the limits of the power to impose conditions on the grant of planning permission following DB Symmetry v Swindon Borough Council?

Answered by: Richard Humpreys QC, "the ideal QC for complex planning cases - clear, concise, commercial - and a joy to work with." He acted in DB Symmetry v Swindon Borough Council.

** Design: When - right now - is good design a reason or material consideration for approval, looking at applications that have turned both ways?

Answered by: Andrew Parkinson, one of the top 5 planning barristers under the age of 35. The Journal of Planning Law included 3 of his cases in its recent summary of important cases.

** Planning Policy: Despite the controversy and hullabaloo, will the government change planning policy over the five-year land supply and housing delivery test?

Answered by: David Brock, who is "immensely experienced, a great communicator and at the forefront of developments." Projects include Canary Wharf and King's Cross. He writes the David Brock Planning Blog.

** Judicial Review: What counts and makes a difference in a successful judicial review of planning decisions, supported by case-law and practical examples?

Answered by: Mark Westmoreland Smith, who acted for Drax in a High Court appeal of development consent given for a Yorkshire power station. He is editor of 'Planning Appeal Decisions; Sweet & Maxwell'.

** Statutory Instruments: Does Gluck v SoS HCLG [2020] mark a departure in the Court's approach to statutory instruments in planning cases?

Answered by: Philippa Jackson, who is "superb at handling heavy technical casework that requires a deep understanding, but yet a simple presentation during advocacy." She acted in Gluck v SoS HCLG.

** Permitted Development: What type of housing is coming forward under permitted development, prior approval and changes to the Use Class order? New and important innovations or damp squibs?

Answered by: Cristina Howick, named as one of the most influential women in UK planning. She wrote the PAS's Advice Note on objectively assessed need and is the Director of Planning Policy for Stantec.

** Viability: What is the state of play on C2/C3 planning distinction, affordable housing and scheme viability following the Rectory Homes case?

Answered by: Sarah Fitzpatrick of Norton Rose, who has advised on some of the UK's most prominent developments, including the US Embassy at Nine Elms, the Shard, the Olympic Park, Crossrail.

To see the full speaker line-up, bios and all 10 thorny questions simply follow the link to the conference home-page where you can also book your ticket online:

https://whitepaper.co.uk/conferences/planning21pd

"Great approach - a wide array of topics that are then drilled down into by each speaker." (Shared by a past-attendee from Ince Gordon Dadds)

You will receive the alumni rate of only £119 plus VAT per person, or £250 plus VAT for your entire team (this is £30 off our regular rates). There is no numbers limit on the team option.

Just reply to this email, and I'll make all the arrangements for you from this end and invoice. Or you can follow the link to the conference home-page where you can book online (your discounted rate is automatically credited through the link):

https://whitepaper.co.uk/conferences/planning21pd

"I found the real-life examples rather than just policy approach most interesting." (Shared by a past-attendee from Boyer Planning)

Since this is a change in how we offer our conferences, we're going to make it risk-free for you. If the quality of the content and the production isn't everything we promise, just let us know. You can receive a full refund up to one month after the event.

This means you can buy your online ticket now safe in the knowledge you have nothing to lose.

If you want to remain capable of offering the most skilful and solution-focused advice to your clients, then this is an opportunity you will not want to miss.

Yours sincerely,

Anna

P.S. Out of 16,590 evaluation forms, 96.53% said "yes," they would recommend White Paper to a colleague.

Anna Linwood
White Paper Conference Company
t. 020 3322 2547
www.whitepaper.co.uk

I sent this message to: 
Tarn.Gascoyne@hertfordshire.gov.uk
No more emails: Simply reply to me letting me know the email addresses you wish to remove and I will take immediate action.

The White Paper Conference Company
Matrix Studios
91 Peterborough Road
London
SW6 3BU


HTML Sample

**Comment by splitbrain** [comment]
Without any exact info which of our mail headers is wrong (and how) we cannot fix this. Please provide more info.
